plume-api/src/posts.rs

```diff
@@ -1,13 +1,10 @@
-use canapi::Endpoint;
-
 #[derive(Clone, Default, Serialize, Deserialize)]
-pub struct PostEndpoint {
-    pub id: Option<i32>,
-    pub title: Option<String>,
+pub struct NewPostData {
+    pub title: String,
     pub subtitle: Option<String>,
-    pub content: Option<String>,
-    pub source: Option<String>,
-    pub author: Option<String>,
+    pub source: String,
+    pub author: String,
+    // If None, a blog for this author will be choosen automatically
```

Comment:
I think it should be chosen automatically only if the user has only one blog, so to be clearly deterministic (currently it's choosing the first found, which was already the old behavior)